### PR TITLE
Install fixes

### PR DESCRIPTION
**Pull request by zaibon**
add some detail in setup.md documentation
small fixed in install-orchestrator.sh script

### CODE REVIEW

File: docs/setup/setup.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ In order to have a full Zero-OS cluster you'll need to perform the following ste
 Create the Docker container with JumpScale9 development environment by following the documentation at https://github.com/Jumpscale/developer#jumpscale-9.
 > **Important:** Make sure you set the `GIGBRANCH` environment variable to 9.0.0 before running `jsinit.sh`. This version of 0-orchestrator will only work with this version of JumpScale.
 
+> **Important:**: Make sure to build the js9 docker with `js9_build -l` and not directly start the docker with `js9_start -b` cause this will not install all the requires librairies.
+
```
Comment (FastGeert) [inline docs/setup/setup.md]: libraries iso librairies

File: docs/setup/setup.md [hunk 2]
```diff
@@ -36,8 +39,11 @@ cd /tmp
 export BRANCH="1.1.0-alpha-3"
 export ZEROTIERNWID="<Your ZeroTier network ID>"
 export ZEROTIERTOKEN="<Your ZeroTier token>"
+export ITSYOUONLINEORG="<itsyou.online organization>"
+export CLIENTSECRET="<client secret of the itsyou.online organization>"
+export DOMAIN="<Your domain name>"
 curl -o install-orchestrator.sh https://raw.githubusercontent.com/zero-os/0-orchestrator/${BRANCH}/scripts/install-orchestrator.sh
-bash install-orchestrator.sh $BRANCH $ZEROTIERNWID $ZEROTIERTOKEN <ITSYOUONLINEORG> <CLIENTSECRET> [<DOMAIN> [--development]]
+bash install-orchestrator.sh $BRANCH $ZEROTIERNWID $ZEROTIERTOKEN <$ITSYOUONLINEORG> <$CLIENTSECRET> [<$DOMAIN> [--development]]
```
Comment (FastGeert) [inline docs/setup/setup.md]: I think we need some more explanation on the new parameters, what they are used for. 

Client secret, is that a JWT?

File: docs/setup/setup.md [hunk 3]
```diff
@@ -51,6 +57,16 @@ tail -s /tmp/install.log
...
 
 ## Setup the AYS configuration service
+
+### Create a JWT token for the AYS CLI:
+Since AYS is protected with JWT, you have to generate a token so the AYS CLI can authenticate to AYS server.
+The CLI provide and easy way to do it:
+```shell
+eval `ays generatetoken --clientid {CLIENT_ID} --clientsecret {CLIENT_SECRET} --organization $ITSYOUONLINEORG`
+```
+CLIENT_ID AND CLIENT_SECRET have to be generated on [Itsyou.online](https://itsyou.online)
```
Comment (FastGeert) [inline docs/setup/setup.md]: Maybe add a link to the itsyou.online documentation to this topic.

File: docs/setup/setup.md [hunk 4]
```diff
@@ -36,8 +38,10 @@ cd /tmp
 export BRANCH="1.1.0-alpha-3"
 export ZEROTIERNWID="<Your ZeroTier network ID>"
 export ZEROTIERTOKEN="<Your ZeroTier token>"
+export ITSYOUONLINEORG="<itsyou.online organization>"
+export DOMAIN="<Your domain name>"
 curl -o install-orchestrator.sh https://raw.githubusercontent.com/zero-os/0-orchestrator/${BRANCH}/scripts/install-orchestrator.sh
-bash install-orchestrator.sh $BRANCH $ZEROTIERNWID $ZEROTIERTOKEN <ITSYOUONLINEORG> <CLIENTSECRET> [<DOMAIN> [--development]]
+bash install-orchestrator.sh $BRANCH $ZEROTIERNWID $ZEROTIERTOKEN <$ITSYOUONLINEORG> <$CLIENTSECRET> [<$DOMAIN> [--development]]
```
Comment (FastGeert) [inline docs/setup/setup.md]: Remove clientsecret

File: docs/setup/setup.md [hunk 5]
```diff
@@ -51,6 +55,16 @@ tail -s /tmp/install.log
...
+
+### Create a JWT token for the AYS CLI:
+Since AYS is protected with JWT, you have to generate a token so the AYS CLI can authenticate to AYS server.
+The CLI provide and easy way to do it:
+```shell
+eval `ays generatetoken --clientid {CLIENT_ID} --clientsecret {CLIENT_SECRET} --organization $ITSYOUONLINEORG`
+```
+CLIENT_ID AND CLIENT_SECRET have to be generated on [Itsyou.online](https://itsyou.online)
+
+### Configuration
```
Comment (FastGeert) [inline docs/setup/setup.md]: Add JWT in configuration blueprint

File: docs/setup/setup.md [hunk 6]
```diff
@@ -51,13 +55,32 @@ tail -s /tmp/install.log
...
+ays generatetoken --clientid {CLIENT_ID} --clientsecret {CLIENT_SECRET} --organization $ITSYOUONLINEORG
+```
+CLIENT_ID AND CLIENT_SECRET have to be generated on [Itsyou.online](https://itsyou.online)  
+From the website, go to your settings, in the `API Keys` panel, generate a new id/secret pair.
+
+This command will output something like:
+```shell
+# Generated Token, please run to use in client:
+export JWT='eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCJ9.eyJhenAiOiJLa0Y0c3IyUll4cXVYWTZlWjVtMWtic0dTbVJRIiwiZXhwIjoxNDk4MTM0MDMyLCJpc3MiOiJpdHN5b3VvbmxpbmUiLCJyZWZyZXNoX3Rva2VuIjoiU2xxLWVfY9ktSjBEalRDbmZPNzA1SDN1ZFN5UyIsInNjb3BlIjpbInVzZXI6bWVtYmVyb2Y6Z3JlZW5pdGdsb2JlLmVudmlyb25tZW50cy5iZS1nOC0zIl0sInVzZXJuYW1lIjoiemFpYm9uIn0.sKVUHPxSb6rxOMx1DKV8w0T0dpyuMya4fBgOV66VFl6-R4p53crvSkHidXRjsKbgbyxV2stsbxV67mo5JPvRN9uaf-pnJ9cXxs74lSq8OoFwre6aG9pG0JPmVt9uMy56'
+```
```
Comment (FastGeert) [inline docs/setup/setup.md]: Is this a real token? Maybe replace a big portion with ...

File: scripts/install-orchestrator.sh [hunk 7]
```diff
@@ -145,15 +141,23 @@ echo "[+] Waiting for AtYourService"
...
-    ays repo create -n orchestrator-server -g js9 >> ${logfile} 2>&1
+    mkdir -p /optvar/cockpit_repos/orchestrator-server >> ${logfile} 2>&1
+    pushd /optvar/cockpit_repos/orchestrator-server
+    mkdir services >> ${logfile} 2>&1
+    mkdir actorTemplates >> ${logfile} 2>&1
+    mkdir actors >> ${logfile} 2>&1
+    mkdir blueprints >> ${logfile} 2>&1
+    touch .ays >> ${logfile} 2>&1
+    git init >> ${logfile} 2>&1
+    git remote add origin https://github.com/zero-os/orchestrator >> ${logfile} 2>&1
```
Comment (FastGeert) [inline scripts/install-orchestrator.sh]: So every install is going to push to this repo?

File: docs/setup/setup.md (hunk 6 above)
Comment (zaibon) [inline docs/setup/setup.md]: It's not real no

File: scripts/install-orchestrator.sh (hunk 7 above)
Comment (zaibon) [inline scripts/install-orchestrator.sh]: push is not a thing yet, so that won't do anything, this is for next release

Comment (FastGeert) [inline scripts/install-orchestrator.sh]: So we could also remove this line?

File: docs/setup/setup.md (hunk 6 above)
Comment (FastGeert) [inline docs/setup/setup.md]: Ok

File: scripts/install-orchestrator.sh (hunk 7 above)
Comment (zaibon) [inline scripts/install-orchestrator.sh]: No cause the code that creates the commit and AYS use the remote info

**Pull request merged**